ntbs-ui-tests/Features/EditNotifications.feature

```diff
@@ -0,0 +1,324 @@
+Feature: Notification editing
+  Happy and error paths for notification creation
+  Notification deletion
+
+  Background: Create new notification
+    Given I have logged in as ServiceUser
+    Given I am on seeded 'TO_BE_EDITED' notification overview page
+
+  Scenario: Edit notification patient details fields
+    When I go to edit the 'PatientDetails' section
+    Then I should be on the PatientDetails page
+    When I enter Jester into 'PatientDetails_GivenName'
+    And I enter Leicester into 'PatientDetails_FamilyName'
+    And I enter 4 into 'FormattedDob_Day'
+    And I enter 8 into 'FormattedDob_Month'
+    And I enter 1932 into 'FormattedDob_Year'
+    And I select radio value 'sexId-2'
+    And I enter 9866698 into 'PatientDetails_LocalPatientId'
+    And I select Chinese for 'PatientDetails_EthnicityId'
+    And I select radio value 'nhs-number-known'
+    And I enter 9452651220 into 'PatientDetails_NhsNumber'
+    And I select Other from input list 'PatientDetails_CountryId'
+    And I enter 1955 into 'year-of-entry'
+    And I enter 56 Schnowzer Boulevard into 'PatientDetails_Address'
+    And I select radio value 'postcode-yes'
+    And I enter ST1 1AA into 'PatientDetails_Postcode'
+    And I make selection 26 from Other section for 'PatientDetails_OccupationId'
+    When I click on the 'save-button' button
+
+    Then I can see the value 'LEICESTER, Jester' for the field 'full-name' in the 'PatientDetails' overview section
```

Comment:
Have you been watching the snooker by any chance?

Reply:
I have, I thought "From Leicester" was too strange a last name so I kept it simple